app/Providers/ModelServiceProvider.php

```diff
@@ -30,13 +34,23 @@ public function boot()
             }
         });
 
+         // When Posts are created reject test events.
+         Post::created(function ($post) {
+            // Check if caption matches 'Test runscope upload' and 'caption_ghost_test'                
+             if (in_array($post->text, ['Test runscope upload', 'caption_ghost_test'])) {
+                 // If it does, run the RejectPost function and delay for 15 minutes 
+                RejectPost::dispatch($post)->delay(now()->addMinutes(15));
```

Comment:
Do you think we could delay for a shorter amount of time (say, 2 minutes)? As long as Runscope or Ghost Inspector would see the same "normal" behavior for the duration of their test run, I think we'd be good (and the less time these sit "unreviewed", the less likely a campaign lead is weirded out by them).

Reply:
That's actually a good idea! I can change that, thanks Dave!

Reply:
If we see any weirdness we can always up the delay!